libraries/core-react/src/hooks/useAutoResize.ts

```diff
@@ -14,13 +14,22 @@ export const useAutoResize = (
         newHeight = Math.max(targetEl.scrollHeight, newHeight)
         if (maxHeight) {
```

Comment:
Is this check necessary as you are already checking it on line 13?

Reply:
`targetEl.style.height = 'auto'` is needed here too, for it to autoresize correctly when user deletes text. It's to rewrite the `height` when it's set before.
 
The nested `maxHeight` check is also needed, as you can see its `if` parent is `!maxHeight || maxHeight > newHeight`, this logic is needed for whenever `maxHeight > newHeight` only

Reply:
Improvements fixed and tested so ready again for review  @mimarz